bioblend/galaxy/libraries/__init__.py

```diff
@@ -117,6 +117,9 @@ def update_library_dataset(self, dataset_id, **kwds):
         :type genome_build: str
         :param genome_build: Replace library dataset genome build (dbkey)
 
+        :type tags: list
+        :param tags: A list of tags to add to the dataset
```

Comment:
Can this also be used to remove tags?

Reply:
Actually it replaces existing tags with the ones given, I will change the docstring

Reply:
(and yes, I'll add tests!)